.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ executors:
       - image: circleci/golang:1.13
     environment:
       - TEST_RESULTS: /tmp/test-results # path to where test results are saved
-      - CONSUL_VERSION: 1.7.0 # Consul's OSS version to use in tests
+      - CONSUL_VERSION: 1.6.3 # Consul's OSS version to use in tests
```

Comment:
todo: we probably want to switch this to 1.7.0 but I have this here now to prove that the tests pass with an older consul version.